client/src/containers/CampaignAdmin/CampaignAdmin.js

```diff
@@ -119,7 +120,8 @@ class CampaignAdmin extends Component {
    * @return {string}
    */
   getActionRoute(id, view) {
-    return `/${this.props.sectionConfig.reactRoutePath}/set/${id}/${view}`;
+    const { reactRoutePath } = this.props.sectionConfig;
+    return joinUrlPaths(`/${reactRoutePath}`, `/set/${id}/${view}`);
```

Comment:
This is possibly better done as 
```    return joinUrlPaths('/', reactRoutePath, `/set/${id}/${view}`);```

I'm not sure if reactRouthPath could have a leading slash or not

Same applies for all the other cases of ````/${reactRoutePath}```` in this PR

Reply:
Done